packages/camel-catalog/assembly/src/main/resources/schemas/KameletConfiguration.json

```diff
@@ -64,18 +64,51 @@
         "default": "",
         "type": "string"
       },
+      "title": "Additional Labels",
       "description": "Map of string keys and values that can be used to organize and categorize (scope and select) objects. May match selectors of replication controllers and services. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels",
-      "type": "object",
-      "title": "Additional Labels"
+      "type": "object"
     },
     "annotations": {
       "additionalProperties": {
         "default": "",
         "type": "string"
       },
+      "title": "Additional Annotations",
       "description": "Annotations is an unstructured key value map stored with a resource that may be set by external tools to store and retrieve arbitrary metadata. They are not queryable and should be preserved when modifying objects. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations",
-      "type": "object",
-      "title": "Additional Annotations"
+      "type": "object"
+    },
+    "kameletProperties": {
+      "title": "Properties",
+      "type": "array",
+      "description": "Configure properties on the Kamelet",
+      "items": {
+        "type": "object",
+        "properties": {
+          "title": {
+            "title": "title",
+            "description": "Display name of the property",
+            "type": "string"
+          },
+          "description": {
+            "title": "description",
+            "description": "Simple text description of the property",
+            "type": "string"
+          },
+          "type": {
+            "title": "type",
+            "description": "JSON-schema type of the property",
+            "type": "string"
+          },
```

Comment:
What do you think of using an enum for this field? With the default value of `string`, similar for the `Kamelet Type`

```json
{
      "title": "Property type",
      "description": "Set the expected type for this property",
      "type": "string",
      "enum": [
        "string",
        "number",
        "boolean"
      ],
      "default": "string"
}
```